tests/ir_check/bundle.expect

```diff
@@ -38,9 +38,9 @@ error: source port does not provide value for as long as destination requires
    ┌─ tests/errors/bundle.fil:15:11
    │
 15 │     out = f{#N+1};
-   │     ----  ^^^^^^^ source is available for @[G+1+#N, G+1+1+#N]
+   │     ----  ^^^^^^^ source is available for @[G+#N+1, G+#N+1+1]
```

Comment:
These changes are a bit suspicious. The original order was like this because the `ExprIdx::add` method reorders the arguments based on their underlying indices.